scene/gui/file_dialog.cpp

```diff
@@ -475,7 +475,7 @@ void FileDialog::update_filters() {
 		String flt=filters[i].get_slice(";",0).strip_edges();
 		String desc=filters[i].get_slice(";",1).strip_edges();
 		if (desc.length())
-			filter->add_item(desc+" ( "+flt+" )");
+			filter->add_item(RTR(desc)+" ( "+flt+" )");
```

Comment:
`RTR(desc)` is incorrect, it's not possible to translate a variable (the translatable strings are extracted from the *source code*).